### DISCUSSION TITLE
Keyboard shortcut to Submit Solution

### DISCUSSION THREAD

**Discussion by Sun-0321** [cpeditor/cpeditor#738] [Q&A] [answered]
Does CP editor have any keyboard shortcut for "submitting to codeforces"?
I checked the key binding and there is no option.

**Comment by ouuan** [comment]
Not in the current version (v6.7).

**Comment by coder3101** [comment, marked as answer]
There is no such shortcut as of now (v6.7) but you can track the progress of this feature request at #201
